src/app/audiologist-navigation/audiologist-navigation.component.ts

```diff
@@ -36,6 +38,7 @@ export class AudiologistNavigationComponent {
     } else {
       this.state.determineState(true, false);
     }
+    this.notesComponent.loadNotes(parseInt(this.patientID));
```

Comment:
You'll need this to prevent login errors.
```suggestion
     if (this.patientID) {
      this.notesComponent.loadNotes(parseInt(this.patientID));
    }
```